frontend/ui/uimanager.lua

```diff
@@ -662,31 +716,31 @@ As far as the actual lifecycle of a widget goes, the rules are:
   this'll ensure that *any* widget including it will be sure that resources are freed when it (or its parent) are closed.
 * Note that there *is* a `Close` event, but it is *only* broadcast (e.g., sent to every widget in the window stack;
   the same rules about propagation apply, but only per *window-level widget*) at poweroff/reboot, so,
-  refrain from implementing custom onClose methods if that's not their intended purpose ;).
+  refrain from implementing custom `onClose` methods if that's not their intended purpose ;).
```

Comment:
There is an other "use case" of the onClose method:
```
frontend/apps/reader/skimtowidget.lua:        self.key_events.Close = { { "Back" }, doc = "close skimto page" }
frontend/apps/filemanager/filemanager.lua:            self.file_chooser.key_events.Close = nil
frontend/ui/widget/bboxwidget.lua:        self.key_events.Close = { {"Back"}, doc = "close windows" }
frontend/ui/widget/virtualkeyboard.lua:        self.key_events.Close = { {"Back"}, doc = "close keyboard" }
frontend/ui/widget/virtualkeyboard.lua:        self.key_events.Close = { {"Back"}, doc = "close keyboard" }
frontend/ui/widget/menu.lua:        self.key_events.Close = { {"Back"}, doc = "close menu" }
frontend/ui/widget/menu.lua:            self.key_events.Close = { {"Left"}, doc = "close menu" }
frontend/ui/widget/configdialog.lua:        self.key_events.Close = { { close_keys }, doc = "close config menu" }
```

Reply:
Ah, that would explain the trend, thanks!